src/fixtures/vite.basic.config.ts

```diff
@@ -5,5 +5,8 @@ import { viteSvgToWebfont } from '../../';
 const webfontFolder = resolve(__dirname, './webfont-test/svg');
 
 export default defineConfig({
+    build: {
+        assetsInlineLimit: 0,
```

Comment:
This line is required to make the tests working. Because the fonts is now a part of the build process, it turns on some of the Vite mechanisms. The fixture font is tiny and Vite inlines it into a resulting CSS. We're preventing this by disabling this mechanism.

Reply:
I think we might actually want to add an additional config to also test inlined CSS, but that definitely is not a blocker 🙂